WordPress/Sniffs/DB/PreparedSQLPlaceholdersSniff.php

```diff
@@ -409,14 +409,30 @@ public function process_token( $stackPtr ) {
 				unset( $match, $matches );
 			}
 
+			/*
+			 * Analyse the query for quoted identifier placeholders.
+			 */
+			$regex = '/`%i`/';
+			if ( preg_match_all( $regex, $content, $matches ) > 0 ) {
+				foreach ( $matches[0] as $match ) {
+					$this->phpcsFile->addError(
+						'Identifier placeholders should not be quoted in the query string in $wpdb->prepare(). Found: %s.',
+						$i,
+						'QuotedIdentifierPlaceholder',
```

Comment:
As I've re-written this test, I've kept `QuotedIdentifierPlaceholder` for now, because it does Simple and Complex placeholder checks (happy to change).

Reply:
Makes sense to me. 👍🏻